fuzz/src/fuzz_targets/codec_bytes.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#![no_main]
+#[macro_use]
+extern crate libfuzzer_sys;
+extern crate tikv;
+
+fuzz_target!(|data: &[u8]| {
+    use tikv::util::codec;
+
+    let _ = codec::bytes::encode_bytes(data);
```

Comment:
Have you considered to remove `let _ = `, or maybe you gonna use it later?

Reply:
I think it is okay to ignore the returned value, as long as we follow rules listed in https://llvm.org/docs/LibFuzzer.html#id22

Reply:
@solotzg  The function result is useless at all, but  i use `let _ = `to handle the warning like this:

> warning: unused `std::result::Result` which must be used
   --> fuzz-helper/src/main.rs:108:13
    |
108 |             buff.encode_duration(&duration);
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: #[warn(unused_must_use)] on by default
    = note: this `Result` may be an `Err` variant, which should be handled